src/app/containers/MPulseBeacon/boomr.js

```diff
@@ -9,6 +9,14 @@ const boomr = (apiKey) => {
   if (window.BOOMR && (window.BOOMR.version || window.BOOMR.snippetExecuted)) {
       return;
   }
+  window.BOOMR_config = {
+    Early: {
+      enabled: true,
```

Comment:
@dr3 what is `enabled: true` doing here?

Reply:
> drew Today at 8:12 AM
> The domain we missed to add to our csp is https://*.akamaihd.net which isnt listed in the CSP section of https://developer.akamai.com/tools/boomerang :disappointed:
> 
> Simon Hearne  10 minutes ago
> Is the akamai plugin disabled?
> 
> drew  9 minutes ago
> Not sure what that is soz :stuck_out_tongue:
> 
> drew  8 minutes ago
> we run > https://github.com/bbc/simorgh/blob/latest/src/app/containers/MPulseBeacon/boomr.js
> 
> Simon Hearne  8 minutes ago
> ```
> window.BOOMR_config = {
> Early: {
>     enabled: true
>     },
> Akamai: {
>     enabled: false
>   }
> };
> ```
> 
> drew  7 minutes ago
> Should we add that?
> 
> Simon Hearne  7 minutes ago
> Yep, that will prevent those requests to akamaihd.net